components/combobox/combobox.jsx

```diff
@@ -459,7 +459,11 @@ class Combobox extends React.Component {
 			}
 		} else if (this.props.isOpen !== nextProps.isOpen) {
 			// eslint-disable-next-line react/no-did-update-set-state
-			this.setState({ isOpen: nextProps.isOpen });
+				this.setState({
```

Comment:
I think we only want to make this `setState` call only when closing the popover, so something like `if (!nextProps.isOpen) { ....`

Reply:
Ok. I can do this more explicit with a dedicated `if` statement.

Reply:
I retested behaviour